.github/workflows/contributor.yml

```diff
@@ -25,16 +29,6 @@ jobs:
         run: |
           pytest --cov=./ --cov-report=xml
 
-  notify:
-    needs: test
-    runs-on: ubuntu-20.04
-    if: contains(fromJson('["refs/heads/main", "refs/heads/binaries-fixes"]'), github.ref) || startsWith(github.ref, 'refs/tags')
-    steps:
-      - name: Slack trigger
-        uses: slackapi/slack-github-action@v1.23.0
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-
```

Comment:
I think the line 51 will still pick up wrong version of python despite us making best effort of setting it correctly using `runs-on: ${{ matrix.os }}`, unless the setting is ignored altogether in both cases we should remove the line:

```
        with:
          python-version: '3.10'
```

Reply:
Removed